lib/evmone/execution_state.hpp

```diff
@@ -216,6 +217,7 @@ class ExecutionState
         deploy_container = {};
         m_tx = {};
         m_initcodes.reset();
+        m_initcodes_validity.clear();
```

Comment:
I think it is better to combine this with `m_initcodes`: to have value as `struct {bytes_view, std::optional<bool>}` or `struct {bytes_view, bool}`.

Reply:
Combined now.